packages/usa-dark-background/src/styles/_usa-dark-background.scss

```diff
@@ -9,14 +9,6 @@
   }
 
   a {
-    color: color($theme-link-reverse-color);
-
-    &:visited {
-      color: color($theme-link-reverse-color);
-    }
-
-    &:hover {
-      color: color($theme-link-reverse-hover-color);
-    }
+    @include set-link-from-bg("base-darker");
```

Comment:
@mejiaj Out of curiosity, does switching from `$theme-link-reverse-color` have any impact on the effectiveness of that theme configuration? Or is that theme configuration being deprecated?

Reply:
Not deprecated, the mixin uses `$theme-link-color` (primary)  and `$theme-link-reverse-color` (base-lighter) by default.